core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -733,4 +729,5 @@ package object config {
       .stringConf
       .toSequence
       .createWithDefault(Nil)
+
```

Comment:
not a big deal at all but I'd remove this line.

Reply:
OK will address.